core/controllers/classroom_test.py

```diff
@@ -71,7 +71,9 @@ def test_get(self):
         topic_services.save_new_topic(admin_id, private_topic)
         public_topic = topic_domain.Topic.create_default_topic(
             topic_id_2, 'public_topic_name', 'abbrev')
-        public_topic.thumbnail_filename = 'Topic.png'
+        public_topic.thumbnail_filename = 'Topic.svg'
```

Comment:
Are we supporting both png as well as svg? If so, we should probably add tests for both?